Snippets/Core/Core_8/Features/MyStartupTaskThatUsesMessageSession.cs

```diff
@@ -14,16 +14,16 @@ class MyStartupTaskThatUsesMessageSession :
     {
         ManualResetEventSlim resetEvent = new ManualResetEventSlim();
 
-        protected override async Task OnStart(IMessageSession session)
+        protected override async Task OnStart(IMessageSession session, CancellationToken cancellationToken = default)
```

Comment:
any reason that this snippet uses `CancellationToken cancellationToken = default` while others use `CancellationToken cancellationToken` ?